tests/features/service-selector.feature

```diff
@@ -0,0 +1,80 @@
+
+
+Scenario: Call setItemsStream with stream of items when Selector instance is empty
+    Given Selector Service with setItemsStream method
+    When  user calls setItemsStream with a stream of items
+    Then  all items are being loaded in the Selector instance
+    And   subscribing to items method returns the stored items
+
+Scenario: Call setItemsStream with a stream of new items when Selector instance is not empty
+    Given  Selector Service with setItemsStream method
+    And    several items are stored in the Selector instance
+    When   user calls setItemsStream with a stream of items that do not exist in the Selector instance
+    Then   the old items are replaced with the new items
+    And    subscribing to items method returns only the new stored ones
+
+ Scenario: Call setItemsStream with a stream of items already stored in the Selector instance
```

Comment:
This scenario shouldn't be merged with the previous one ?
The aim of of these 2 scenarios are the same: checking that something we input in the selector through `setItemsStream` is stored, no matter if it is a different stream or the same